engine/runtime/src/test/java/org/enso/interpreter/test/LazyAtomFieldTest.java

```diff
@@ -0,0 +1,102 @@
+package org.enso.interpreter.test;
+
+import java.io.ByteArrayOutputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.enso.polyglot.RuntimeOptions;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Language;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.Value;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import org.junit.Before;
+import org.junit.Test;
+
+public class LazyAtomFieldTest {
```

Comment:
If possible, please, try to use `TestBase` as much as possible, adding some methods there if necessary. I can see a lot of code duplication here, like `prepareCtx`, and `evalCode`.

Reply:
I don't mind if there is duplicated code in tests. I even blogged about [never update (API) test!](http://wiki.apidesign.org/wiki/Never_update_tests) once. If one keeps backward compatibility, old and duplicated tests are in fact good.

I tried to use the suggested helper methods, but `prepareCtx` misses support for logging and `evalCode` does something completely different than my method. At least I can use `MethodNames`: 4ad5130dab2